### DISCUSSION TITLE
multiple emails for a single user

### DISCUSSION THREAD

**Discussion by LORE-D-NATO** [supabase/supabase#28926] [Questions]
For my project I need users to be able to add additional emails to their account, similar to the way GitHub does it. My understanding is that this isn't supported in the current auth system, so I'd appreciate any suggestions for a possible workaround!

**Comment by silentworks** [comment]
As you've mentioned this isn't supported by Supabase Auth. Until it is there isn't a way to do this if you require the user to be able to sign in with either email address.
